content/en/references/configuration.md

```diff
@@ -498,6 +498,8 @@ $ python -m localstack.cli.main --profile=dev config show
 
 {{< callout >}}
 The `CONFIG_PROFILE` is a CLI feature and cannot be used with a Docker/Docker Compose setup.
-You can look at [alternative means of setting environment variables](https://docs.docker.com/compose/environment-variables/set-environment-variables/) for your Docker Compose setups.
+
 For Docker setups, we recommend passing the environment variables directly to the `docker run` command.
+
+You can also set environment variables in a file and pass it to the LocalStack Docker container by using the `--compose-env-file` flag.
```

Comment:
I think there is a misunderstanding how the `--compose-env-file` works. It only works in `LAMBDA_DOCKER_FLAGS`, `ECS_DOCKER_FLAGS` and the likes.

It is not a command line parameter for the CLI, nor for the docker CLI, nor will it work for the regular `DOCKER_FLAGS` which are used for starting the LocalStack container itself. It is purely a flag for container localstack starts on its own.